### PR TITLE
BOTMETA.yml: update team_virt

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
https://github.com/ansible-collections/community.general/pull/1949#issuecomment-822471204

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
BOTMETA.yml

### CODE REVIEW

**Comment by felixfontein** [comment]
@Ajpantuso can you please approve this PR to confirm?

@joshainglis @karmab @tleguern @Thulium-Drake if you do not want to be informed on changes on the proxmox inventory plugin, please write here!

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -1017,4 +1019,4 @@ macros:
   team_scaleway: QuentinBrosse abarbare jerome-quere kindermoumoute remyleone sieben
   team_solaris: bcoca fishman jasperla jpdasma mator scathatheworm troy2914 xen0l
   team_suse: commel dcermak evrardjp lrupp toabctl AnderEnder alxgu andytom
-  team_virt: joshainglis karmab Aversiste Thulium-Drake
+  team_virt: joshainglis karmab Aversiste Thulium-Drake Ajpantuso
```
Comment (felixfontein) [inline .github/BOTMETA.yml:1022]:
```suggestion
  team_virt: joshainglis karmab tleguern Thulium-Drake Ajpantuso
```
While we're at it... :) CC @tleguern

**Review by Ajpantuso — APPROVED** [review]
LGTM, thanks

**Comment by felixfontein** [comment]
@Andersson007 I'm afraid you need to rebase this PR.

File: .github/BOTMETA.yml (hunk 1 above)
Comment (Andersson007) [reply .github/BOTMETA.yml:1024]: I can't see who put the thumbs up:) If it was Tristan, feel free to commit and merge:)

**Comment by Andersson007** [comment]
@felixfontein done, thanks

File: .github/BOTMETA.yml (hunk 1 above)
Comment (felixfontein) [reply .github/BOTMETA.yml:1024]: @Andersson007 it was you ;)

Comment (Andersson007) [reply .github/BOTMETA.yml:1024]: hahahah:)

**Comment by felixfontein** [comment]
@Ajpantuso thanks!

File: .github/BOTMETA.yml (hunk 1 above)
Comment (tleguern) [reply .github/BOTMETA.yml:1024]: I tried to reply by email but I apparently messed it enough to fail. So yes, I renamed my Github account and I have not yet propagated the change where it needs.

**Comment by felixfontein** [comment]
Manual backport to stable-1 in #2306.

**Pull request merged**